src/main/webapp/stats.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Stats</title>
+        <link rel="stylesheet" href="/css/main.css">
+        <link rel="stylesheet" href="/css/stats.css">
+        <script src="/js/stats-loader.js"></script>
+    </head>
+    <body onload="buildUI()">
```

Comment:
We can leave this as-is for the purpose of this change, but it seems to me that the approach taken to loading the JavaScript and issuing this fetch could be more efficient...

Right now, you are waiting until the `<body>` has finished loading to issue the call to `/stats`, but you could actually begin to fetch `/stats` fetch immediately and make it so that only the `.then` piece which puts the result into the page needs to wait for the page to finish loading.